Automattic/sensei
# Importer: Delete uploaded files

### Changes proposed in this Pull Request

* Adds the ability to delete uploaded files.
* Minor refactor to prevent passing `jobId` without putting it in the actions.
* Note: This wasn't in the sketches, but we thought it would be useful so people can reset the importer state.

### Testing instructions

* Build assets.
* In Sensei LMS > Importer, upload files on the first step.
* Once you've uploaded, click on the bin next to the file name to remove the file.
* Observe the file upload state get reset.
* Check the database and make sure the uploaded file attachment has been removed.

<img width="740" alt="Screen Shot 2020-06-30 at 3 27 08 pm" src="https://user-images.githubusercontent.com/68693/86138318-37d26080-bae6-11ea-9a73-17e1e5b33f98.png">
